### PR TITLE
Created beacon config template, added bentoctl tool to copy files for beacon, katsu

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- beacon config & cohort moved to `etc/templates/beacon` as templates
- `lib/beacon/beacon_config.json` & `becaon_cohort.json` are git ignored
- added cli tool init-config _service_ for beacon and katsu to copy the example configs to their respective locations

### CODE REVIEW

**Review by v-rocheleau — COMMENTED** [review]
Looks good, should this also be done for `beacon_cohort.json` while we are here @gsfk ?

**Comment by gsfk** [comment]
> Looks good, should this also be done for `beacon_cohort.json` while we are here @gsfk ?

Yes, good catch, `beacon_cohort.json` should be handled in the same way. Hopefully soon we can eliminate this file entirely.

**Comment by gsfk** [comment]
Just starting tests, now, first comment is that it probably needs some README additions

**Review by gsfk — CHANGES_REQUESTED** [review]
Okay, I guess the intended command for beacon is 

`./bentoctl.bash init-config beacon`

and simiilarly for katsu. This should be in the README somewhere. I wonder also if there's a use case for just `./bentoctl.bash init-config` that does katsu and beacon together. I guess this will come anyway when we manage to unify the config files. 

Otherwise this looks great, everything works as expected.

File: py_bentoctl/other_helpers.py [hunk 1]
```diff
@@ -620,6 +621,36 @@ def convert_phenopacket_file(source: str, target: str):
...
+    config_dest_path = (dest_dir / "beacon_config.json")
+
+    cohort_template_path = (template_dir / "beacon_cohort.example.json")
+    cohort_dest_path = (dest_dir / "beacon_cohort.json")
+
+    _file_copy(config_template_path, config_dest_path, force)
+    _file_copy(cohort_template_path, cohort_dest_path, force)
+
+
+def _init_katsu(force: bool):
```
Comment (davidlougheed) [inline py_bentoctl/other_helpers.py:646]: same here

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: docs/installation.md [hunk 2]
```diff
@@ -143,9 +143,18 @@ Bento's clinical/phenotypic metadata service:
 ```bash
 # public service configuration file. Required if BENTOV2_USE_BENTO_PUBLIC flag is set to `1`
 # See Katsu documentation for more information about the specifications
-cp ./etc/katsu.config.example.json ./lib/katsu/config.json
+./bentoctl.bash init-config katsu
 ```
 
+### Beacon configuration
```
Comment (davidlougheed) [inline docs/installation.md:149]: two newlines between `###` blocks

**Review by davidlougheed — CHANGES_REQUESTED** [review]

**Review by davidlougheed — APPROVED** [review]
lgtm

**Comment by gsfk** [comment]
`./bentoctl.bash init-config beacon` gives me permission denied error, since config directory was created by root

**Review by gsfk — APPROVED** [review]

**Pull request merged**